docs/guidelines/git-github.md

```diff
@@ -0,0 +1,306 @@
+git remote -v
+```
+
+should out output remote name and a URI containing the `git@github.com`
+
+```
+origin	git@github.com:equinor/appsec.git (fetch)
+origin	git@github.com:equinor/appsec.git (push)
+```
+
+- You can add and change remotes for a cloned repo, from HTTPS to SSH using the command `git remote set-url <remote name> <new uri>`
+
+### Configure the SSH key for usage with Equinor's SSO protected resources
+
+The Equinor organization on github.com is protected behind SSO login. In order for your SSH key to be used with resources in the Github "Equinor" or "Equinor-Playground" organizations you need to authorize the key for these permissions on your behalf. [Github documentation](https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/authorizing-an-ssh-key-for-use-with-saml-single-sign-on) gives you all the details.
+
+- To authorize your key head over to your [SSH Key Settings](https://github.com/settings/keys) on your profile on Github, select the key in question, select "Configure SSO" and authorize against organisation(s).
+
+### Rotating SSH keys
+
+!!! Tip "Rotate your SSH keys"
+
+    Your SSH keys, and passphrases, should be rotated at least on a yearly basis. Put an re-occurring appointment in your calender for this. This process could be automated, but doing it once in a while in manual mode may help you not to forget how things work.
+
+## Github.com
+
+### Basic config for your account
+
+You will find your Github settings at [https://github.com/settings/profile](https://github.com/settings/profile).
+
+We recommend the following settings:
+
+#### [Public profile](https://github.com/settings/profile)
+
+-   Use your full name
+-   Select an recognizable avatar/picture for profile picture
+
+
+#### [Emails](https://github.com/settings/emails)
+
+- Have a primary and a secondary email address registered
+- Select "Keep my email addresses private"
+- Select "Block command line pushed that expose my email"
+
+#### [Password and authentication](https://github.com/settings/security)
+
+- Enable Two-Factor authentication
+  - This is **not** the Two-Factor for your Equinor account, this is for your account at github.com
+- Enable multiple Two-Factor methods
+  - Authenticator App
+  - SMS/Text
+- Preferred 2FA method should be Authenticator app
+- (Experiment with PassKeys or Security Keys)
+- Extract "Recovery codes" and store in your password manager
+
+#### [Codespaces](https://github.com/settings/codespaces)
+
+- Enable GPG verification
+- Validate the "Editor preference"
+  - "Visual Studio Code for the Web" is a good option for many
+- Validate "Default retention period"
+  - 14 days may be a good period (Codespaces incurs storage charges)
+- Region
+  - Set manually to "Europe West"
+
+#### [Code security an analysis](https://github.com/settings/security_analysis)
+
+- "Push protection for yourself" should be enabled
+  - Block push with supported secrets against public repos
```

Comment:
I got a bit confused on this one, while reviewing in the codespaces it looks just like an additional bullet point not a sub bullet point of the one above, spent some time realizing that this was not on the page for me to enable 🙈

Reply:
Thanx. The indenting for mkdocs was a bit "unusual", it required two "tabs", so it was a problem for multiples sections